modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -528,9 +528,6 @@ export default class ParticipantConnectionStatusHandler {
             remoteTrack.off(
                 JitsiTrackEvents.TRACK_MUTE_CHANGED,
                 this._onSignallingMuteChanged);
-            remoteTrack.off(
```

Comment:
We don't need to remove the listener?

Reply:
To encapsulate, yes. Practically and specifically for jitsi-meet, no, because jitsi-meet calls to remove all listeners. This is why I offered alternatives, and this was the "easiest" alternative.

Reply:
I agree this is the easiest way but shouldn't we try to remove the listener?

Reply:
if we want to remove the listener I find the 2nd alternative most appealing because it avoids introducing extra state variables; I'm also happy to implement the 1st one or leave this PR as is. @hristoterezov do you have a preference?

Reply:
No